### DISCUSSION TITLE
为什么我调用不了chatgpt

### DISCUSSION THREAD

**Discussion by an2058185378** [dmMaze/BallonsTranslator#495] [Q&A]

**Comment by dmMaze** [comment]
你用的中转返回不规范，另外最好别把带 api key 的截图发上来

**Reply by an2058185378** [reply]
临时令牌没事 另外可以问一下什么事是中转返回吗 抱歉第一次弄这个没听懂

**Reply by dmMaze** [reply]
> 临时令牌没事 另外可以问一下什么事是中转返回吗 抱歉第一次弄这个没听懂

就是这个中转发回的结果和 openai 官方的结果格式不一样

**Reply by an2058185378** [reply]
api url那栏吗 抱歉捣鼓了一下还是没懂这个不是直接填第三方api的中转地址的吗

**Reply by dmMaze** [reply]
> api url那栏吗 抱歉捣鼓了一下还是没懂这个不是直接填第三方api的中转地址的吗

这是这个中转站的问题，你只能换一个或者用官方的，或者给让他们规范一下返回

**Reply by an2058185378** [reply]
好的 谢谢啦
